### PR TITLE
Add December (the very first) MTC (magnanimous-turbo-chainsaw) release

### PR DESCRIPTION
**Pull request by tonytan4ever**

### CODE REVIEW

**Comment by tonytan4ever** [comment]
:shipit:

**Comment by tonytan4ever** [comment]
:shipit:

**Comment by tonytan4ever** [comment]
recheck_container_lint_test

**Comment by tonytan4ever** [comment]
:shipit:

File: components/rpc-mtc.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 artifact_stores: []
 is_product: false
 name: rpc-mtc
```
Comment (hughsaunders) [inline components/rpc-mtc.yml]:
```suggestion
name: magnanimous-turbo-chainsaw
```

Comment (hughsaunders) [reply components/rpc-mtc.yml]: This was fixed in #103

**Comment by hughsaunders** [comment]
:shipit:

**Comment by hughsaunders** [comment]
:shipit:

**Comment by hughsaunders** [comment]
:shipit:

**Comment by hughsaunders** [comment]
:shipit:

**Comment by hughsaunders** [comment]
Failed on wiki update (https://rpc-openstack.atlassian.net/browse/RE-2238)

**Comment by hughsaunders** [comment]
Release has been created, so I'm going to merge this.

**Pull request merged**